docs/running-on-yarn.md

```diff
@@ -304,6 +304,14 @@ If you need a reference to the proper location to put log files in the YARN so t
   </td>
 </tr>
 <tr>
+  <td><code>spark.yarn.attemptFailuresValidityInterval</code></td>
+  <td>-1</td>
+  <td>
+  Ignore the failure number which happens out the validity interval (in millisecond).
```

Comment:
I don't really feel strongly one way or another. I just think putting it in the `.am` namespace makes it clear that it's a setting that affects the AM behavior, and not other parts of the app.